packages/compass-aggregations/src/components/settings/settings.jsx

```diff
@@ -76,11 +77,11 @@ class Settings extends PureComponent {
       return (
         <div className={classnames(styles['input-group'])}>
           <div className={classnames(styles['input-meta'])}>
-            <label htmlFor='aggregation-limit'>Limit</label>
-            <p id="aggregation-limit-description">
+            <Label htmlFor='aggregation-limit'>Limit</Label>
+            <Description id="aggregation-limit-description">
```

Comment:
Before/After of this page:

<img width="500" alt="Screen Shot 2022-03-21 at 3 16 51 PM" src="https://user-images.githubusercontent.com/1791149/159352680-051c532c-412e-4dcf-8b16-4f1be53aa132.png">
<img width="500" alt="Screen Shot 2022-03-21 at 3 16 39 PM" src="https://user-images.githubusercontent.com/1791149/159352678-a7b90610-4a18-4387-9a05-0f75c340a794.png">